tests/aims/test_flows/test_anharmonic_quantification.py

```diff
@@ -0,0 +1,329 @@
+import os
+
+import numpy as np
+import pytest
+from jobflow import run_locally
+from pymatgen.io.aims.sets.core import SocketIOSetGenerator, StaticSetGenerator
+
+from atomate2.aims.flows.anharmonicity import AnharmonicityMaker
+from atomate2.aims.flows.phonons import PhononMaker
+from atomate2.aims.jobs.core import RelaxMaker, StaticMaker
+from atomate2.aims.jobs.phonons import (
+    PhononDisplacementMaker,
+    PhononDisplacementMakerSocket,
+)
+
+cwd = os.getcwd()
```

Comment:
isn't there another option to get the current directory in test environment?

Reply:
There might be, I think this was probably copied from some other FHI-aims tests and I don't know if Andrei and I thought to check when we first wrote it

Reply:
I thin the `clean_dir` fixture takes care of this: https://github.com/materialsproject/atomate2/blob/a9e5edecc267760c84562efe8101acc7390b61cc/tests/conftest.py#L41
I think everywhere else in the code, this is used for the workflow tests